windows/utils/win_enable_vbs_guest.yml

```diff
@@ -35,7 +35,8 @@
       reg add "HKLM\SYSTEM\CurrentControlSet\Control\Lsa" /v "LsaCfgFlags" /t REG_DWORD /d 1 /f
   when: >
     (guest_os_build_num | int < 22621) or
-    (guest_os_product_type | lower == 'client' and guest_os_build_num | int >= 22621 and guest_os_edition | lower not in ['enterprise', 'education'])
+    (guest_os_product_type | lower == 'client' and guest_os_build_num | int >= 22621 and guest_os_edition | lower not in ['enterprise', 'education']) or
```

Comment:
Sign-off is missing. BTW, this line is too long, you can split it into 3 lines since it is in parentheses.